Dockerfile.template

```diff
@@ -49,4 +49,7 @@ RUN poetry config settings.virtualenvs.create false \
 # Add binaries' path to PATH.
 ENV PATH="/murakami/bin:${PATH}"
 
-ENTRYPOINT [ "python", "-m", "murakami" ]
+# Mount the expected system path for saving data
+VOLUME ["/data/", "/var/lib/murakami"]
```

Comment:
It sounds like you're trying to map a `/data` volume to `/var/lib/murakami`, but this line will actually just declare both `/data` and `/var/lib/murakami` as exportable volumes. I often like to declare important data directories as volumes in my Dockerfiles in case I might want to connect other containers to them, but it's not strictly necessary and in this case I don't think it will accomplish what you're aiming for.

Reply:
Fine with going with your recommendation. The reason I put this in was related to your comment about storing `config.json` for WT. And we are already using env vars or the .toml values to point exporters at `/data/`